lambda_functions/crud/operations.py

```diff
@@ -332,6 +379,85 @@ def get_all_review_questions_db():
     return review_questions
 
 
+def give_experience_point(user_id):    
+    user = get_user_by_id(user_id)
+    user.experience_points = user.experience_points + 1
+    if user.experience_points >= 5:
+        user.level = 2
+    update_object_db(user)
+
+def check_if_review_still_needed(item_id, is_peer_review):
+    item = get_item_by_id(item_id)
+    status = item.status
+    if is_peer_review == True:
+        if status == "needs_senior":
+            return True
+        else:
+            return False
+    if is_peer_review == False:
+        if status == "needs_junior":
+            return True
+        else:
+            return False
+
+def close_open_junior_review(item_id, peer_review_id):
+    """Returns all reviews from the database that belong to the item with the specified id
+    
+    Parameters
+    ----------
+    item_id: Str, required
+        The item id to query for
+    
+    Returns
+    ------
+    review: Review
+        The open junior review
+    """
+    session = get_db_session()
+    query_result = session.query(Review).filter(
+        Review.item_id == item_id,
+        Review.is_peer_review == "false",
+        Review.peer_review_id == None
+        )
+    open_junior_review = query_result.one()
+    open_junior_review.peer_review_id = peer_review_id
+    update_object_db(open_junior_review)
+
+def get_pair_difference(review_id):
+    junior_review = get_review_by_peer_review_id_db(review_id)
+    peer_review_answers = get_review_answers_by_review_id_db(review_id)
+    junior_review_answers = get_review_answers_by_review_id_db(junior_review.id)
+
+    junior_review_score_sum = 0
+    peer_review_score_sum = 0
+
+    for answer in junior_review_answers:
+        junior_review_score_sum = junior_review_score_sum + answer.answer
+
+    for answer in peer_review_answers:
+        peer_review_score_sum = peer_review_score_sum + answer.answer
+    
+    junior_review_average = junior_review_score_sum / junior_review_answers.count()
+    peer_review_average = peer_review_score_sum / peer_review_answers.count()
+
+    difference = abs(junior_review_average - peer_review_average)
```

Comment:
I think we would maybe get a more accurate result if we computed the difference between review and peer review for each individual question. What do you think?

Reply:
I believe, that concerning the final result, it would not make a difference. At least if my math is correct.

(a1 + a2 + ... + an) / n - (b1 + b2 + ... + bn) / n <-- My approach
= a1/n + a2/n + ... + an/n - b1/n - b2/n - ... - bn/n
= (a1 + a2 + ... + an - b1 - b2 - ... -bn) / n
= ((a1 - b1) + (a2 - b2) + ... + (an - bn)) / n <-- Your proposal

Reply:
Let's have the concept team discuss this potential issue in more detail in the next sprint.